src/createServer.js

```diff
@@ -1,3 +1,50 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase/convertToCase');
+const {
+  NO_TEXT_ERR_MESSAGE,
+  NO_CASE_ERR_MESSAGE,
+  NO_AVAILABLE_CASE_ERR_MESSAGE,
+  AVAILABLE_CASES,
+} = require('./variables');
+
+function createServer() {
+  const server = http.createServer((req, res) => {
+    const normalizedUrl = new URL(req.url, 'http://localhost:5700/');
```

Comment:
It is better to store the `port` as a separate constant for more convenient control.